src/components/notifications/NotificationsMenu.js

```diff
@@ -156,7 +162,15 @@ function NotificationsMenu(props) {
                 }
                 setError(null);
             },
-            onError: setError,
```

Comment:
I noticed this is still in a `Draft`, but in case this got missed, I just wanted to point out that this was the only place where an error was getting set with `setError`.

So now that it's replaced here with `setErrorObject`, the `error` state should probably also be refactored everywhere to use `errorObject` instead (or refactor your new code to reuse `error` and `setError` instead).